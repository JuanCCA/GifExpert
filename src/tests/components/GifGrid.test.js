import {shallow} from 'enzyme';
import {GifGrid} from '../../components/GifGrid'
import React from 'react';
import { useFetchGifs } from '../../hooks/useFetchGifs';
jest.mock('../../hooks/FetchGifs')


describe('Tests for GifGrid', () => {
    

    test('should create the component correctly', () => {
        


        const wrapper = shallow( <GifGrid category="Coñaroo"/>);

        expect(wrapper).toMatchSnapshot();


    })

    test('should show items when images ar loaded by useFetchGifs', () => {
        
        
       

        const wrapper = shallow( <GifGrid category="Coñaroo"/>);



    })
    



})
