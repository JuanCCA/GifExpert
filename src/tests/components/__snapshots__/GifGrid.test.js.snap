// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for GifGrid should create the component correctly 1`] = `
<Fragment>
  <h3>
    Co√±aroo
  </h3>
  <p>
    Loading
  </p>
  <div
    className="card-grid"
  />
</Fragment>
`;
